lib/tasks/asset_sync.rake

```diff
@@ -10,21 +9,4 @@ namespace :assets do
       AssetSync.clean
     end
   end
-
-end
-
-if Rake::Task.task_defined?("assets:precompile:nondigest")
```

Comment:
@cameronjacoby I'm testing this with my own S3 bucket and AWS access key credentials, and I can confirm that with this branch we have to call an explicit `rake assets:sync`, which isn't a problem, but something we should be aware of.

And the double downer is that it doesn't actually work as expected. I see an "assets" directory in my S3 bucket, but no "packs" directory.